docs/en/changes/changes.md

```diff
@@ -5,9 +5,12 @@
 
 #### OAP Server
 
+Add 133 134 component ID for impala jdbc java plugin.
 
 #### UI
 
+Add impala icon for impala jdbc java plugin.
```

Comment:
```suggestion
```

I don't think you have added the icon to UI.

Reply:
ui repo, https://github.com/apache/skywalking-booster-ui/pull/153

Reply:
Then we need to wait for that PR merged, and you update the UI submodule in this PR.

Reply:
Okey.

Reply:
UI PR is merged, please update the git submodule.